docs/src/theme/MDXComponents.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
```

Comment:
@petercrocker do we need to add this component to Infrahub-docs as well ?

Reply:
I think it's already there: https://github.com/opsmill/infrahub-docs/blob/894076746c6fa744dd1eca674d6eb25ccb1d95f0/docs/package.json#L27

Reply:
Sorry I commented the wrong line in this file
I meant `"@iconify/react"` and the Icon component